content/basics/navigating.md

```diff
@@ -4,4 +4,20 @@ date =  2017-10-30T10:09:25-04:00
 weight = 6
 +++
 
-Navigating basics.
+Let's start with a little history. Why does vim use **hjkl** keys to navigate?  
+When Bill Joy created the *Vi* he used the ADM-3A terminal, as you can see the arrows are *hjkl* keys. Then he used the same keys...  
```

Comment:
Let's start with a little bit of history. When Bill Joy created the original *Vi* editor in 1976 he used ADM-3A terminal. As you can see the arrow keys are in the position of the *hjkl* keys. He then used the same keys for navigation.